src/envpak.cpp

```diff
@@ -462,9 +471,9 @@ int CheckFP( double v, const char* vTag)		// check for valid FP value
 // see CHECKFP macro (cnglob.h)
 // returns 0 if OK, else 1
 {	const char* t;
-	if (_isnan( v))
+	if (isnan( v))
```

Comment:
shouldn't we use the std namespace here if we're using it below?

Reply:
I was wondering the same thing. Is there an equivalent C function that is being used in this case? Probably can't hurt to use `std::`.